v1.0-beta/addressbook.minified.json

```diff
@@ -133,6 +139,18 @@
       "lastUpdated": "Thu, 28 Sep 2023 14:09:55 GMT",
       "contractType": "AffinePass",
       "abiPath": "v1.0-beta/137/AffinePass.json"
+    },
+    "AffinePassBridge": {
```

Comment:
Renaming it to `AffinePassBridgeEthereum` will have similar naming convention.

Reply:
done